modules/calendar/add_edit_event.php

```diff
@@ -55,10 +55,22 @@
 
 /* Things that might be passed by our opener. */
  $eid           = $_GET['eid'];         // only for existing events
- $date          = $_GET['date'];        // this and below only for new events
- $userid        = $_GET['userid'];
- $default_catid = $_GET['catid'] ? $_GET['catid'] : '5';
- //
+ $date          = $_GET['date'];        // this for new events
+ $userid        = $_GET['userid'];      // this for new events
+ if ($_GET['catid']) {
+    // when existing event
+    $default_catid = $_GET['catid'];
+ } else {
+  // when new event
+  if ($_GET['prov']) {
+    // selecting provider tab
+    $default_catid = '2'; // In Office as default category
```

Comment:
We really should link these to variables saved...probably in the category table where a row is marked as the clinic default....or maybe in the user prefs table for default category.  This would avoid issues that happen when all the default categories are changed (or an existing database is used) that has totally different indexes.

Reply:
I *think* there is somewhere else that sets a calendar category as default, at least at the calendar level...or there used to be.  Maybe in the practice/calendar/Category creation UI?  Probably best to look at the table for a value.